### PR TITLE
Fix GH-806: Teacher Account Video Embded

### LINKED ISSUE #806: Add Teacher Account video in-line on FAQ page
Amend the first question and embed the video directly after the paragraph of text: 

**What are teacher accounts?**
A Scratch Teacher Account provides teachers and other educators with additional features to manage student participation on Scratch, including the ability to create student accounts, organize student projects into studios, and monitor student comments. Learn more about Teacher Accounts in the video below:

[Embed Video]

### PR DESCRIPTION
**Pull request by St19Galla**
This PR fixes #806 by embedding the youtube video using `iframe` and amending the text to what was suggested by @kaschm :)

This is what the page looks like for me after the PR on my local host:
screen shot 2016-08-09 at 10 21 11 am

Hope this helps!
@thisandagain @mewtaylor @rschamp @technoboy10

### CODE REVIEW

File: src/views/teachers/faq/faq.jsx [hunk 1]
```diff
@@ -20,6 +20,8 @@ var TeacherFaq = injectIntl(React.createClass({
                         <dl>
                             <dt><FormattedMessage id='teacherfaq.teacherWhatTitle' /></dt>
                             <dd><FormattedHTMLMessage id='teacherfaq.teacherWhatBody' /></dd>
+                            <iframe width="565" height="318" src="https://www.youtube.com/embed/7Hl9GxA1zwQ"
+                            frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen>...</iframe>
```
Comment (rschamp) [inline src/views/teachers/faq/faq.jsx]: Thanks so much for helping out with this @St19Galla! Would you please update the `<iframe>` here to not include the `...`?

File: src/views/teachers/faq/faq.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ var TeacherFaq = injectIntl(React.createClass({
                             <dt><FormattedMessage id='teacherfaq.teacherWhatTitle' /></dt>
                             <dd><FormattedHTMLMessage id='teacherfaq.teacherWhatBody' /></dd>
                             <iframe width="565" height="318" src="https://www.youtube.com/embed/7Hl9GxA1zwQ"
-                            frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen>...</iframe>
+                            frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```
Comment (rschamp) [inline src/views/teachers/faq/faq.jsx]: Thanks!

**Comment by St19Galla** [comment]
@rschamp Ah, sorry about that! I noticed that most of the other embedded videos on the website included the `...` so I thought it wouldn't be good here too. My bad! :)

**Comment by rschamp** [comment]
Oh, I didn't realize that. Maybe it's something YouTube adds in their embed code. Thanks for your help!

**Pull request merged**